corehq/motech/openmrs/tasks.py

```diff
@@ -225,17 +225,15 @@ def import_patients_to_domain(domain_name, force=False):
                 owner = CommCareUser.get(importer.owner_id)
             except ResourceNotFound:
                 logger.error(
-                    'Project space "{domain}" has no user to own cases imported from OpenMRS Importer '
-                    '"{importer}"'.format(
-                        domain=domain_name, importer=importer)
+                    f'Project space "{domain_name}" has no user to own cases imported from OpenMRS Importer '
+                    f'"{importer}"'
                 )
                 continue
             import_patients_of_owner(requests, importer, domain_name, owner)
         else:
             logger.error(
-                'Error importing patients for project space "{domain}" from OpenMRS Importer "{importer}": Unable '
-                'to determine the owner of imported cases without either owner_id or location_type_name'.format(
-                    domain=domain_name, importer=importer)
+                f'Error importing patients for project space "{domain_name}" from OpenMRS Importer "{importer}": '
+                f'Unable to determine the owner of imported cases without either owner_id or location_type_name'
```

Comment:
nit: this doesn't need to be an `f` string since it has no substitutions
```suggestion
                'Unable to determine the owner of imported cases without either owner_id or location_type_name'
```